discord/message.go

```diff
@@ -147,13 +147,14 @@ const (
 	// name of the thread.
 	ThreadCreatedMessage
 	InlinedReplyMessage
-	ApplicationCommandMessage
+	ChatInputCommandMessage
```

Comment:
Is ChatInputCommandMessage the new ApplicationCommandMessage? Why the rename?

Reply:
Yes. It was renamed when the context menu command message was added to the docs.
https://github.com/discord/discord-api-docs/commit/3973676f437a21714d15338ccab9585c2492847f